test/e2e/scheduling/limit_range.go

```diff
@@ -206,10 +205,8 @@ var _ = SIGDescribe("LimitRange", func() {
 		framework.ExpectNoError(err)
 
 		ginkgo.By("Verifying the LimitRange was deleted")
-		gomega.Expect(wait.Poll(time.Second*5, e2eservice.RespondingTimeout, func() (bool, error) {
-			selector := labels.SelectorFromSet(labels.Set(map[string]string{"name": limitRange.Name}))
-			options := metav1.ListOptions{LabelSelector: selector.String()}
-			limitRanges, err := f.ClientSet.CoreV1().LimitRanges(f.Namespace.Name).List(context.TODO(), options)
+		err = wait.Poll(time.Second*5, e2eservice.RespondingTimeout, func() (bool, error) {
+			limitRanges, err := f.ClientSet.CoreV1().LimitRanges(f.Namespace.Name).List(context.TODO(), metav1.ListOptions{})
```

Comment:
why the list options was removed, should we be listing the specific limit range in question?

Reply:
In theory, it's fine to just check the specific LimitRange is deleted (by checking if the error is of type APINotFound). The current style (to ensure no LimitRange in the namespace) don't need to check the returned error, and also can avoid the case that any other LimitRange gets created during the test.